test/JikanTest/Helper/JStringTest.php

```diff
@@ -11,6 +11,32 @@
  */
 class JStringTest extends TestCase
 {
+    /**
+     * @test
+     */
+    public function it_checks_for_string_float()
+    {
+        self::assertEquals(
+            true,
+            JString::isStringFloat('3.123')
+        );
+
+        self::assertEquals(
+            true,
+            JString::isStringFloat(' 3.123')
+        );
+
+        self::assertEquals(
+            false,
+            JString::isStringFloat(' abc 3.123')
+        );
+
+        self::assertEquals(
+            false,
+            JString::isStringFloat('3..123')
+        );
+    }
```

Comment:
Fyi: https://phpunit.de/manual/3.7/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers